tests/test_wave_03.py

```diff
@@ -59,7 +59,7 @@ def test_friends_unique_movies_not_duplicated():
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
```

Comment:
What could we assert to make sure the `friends_unique_movies` is holding the correct 3 movies?